src/modules/mc_pos_control/multicopter_position_mode_params.c

```diff
@@ -35,12 +35,15 @@
  * Position/Altitude mode variant
  *
  * The supported sub-modes are:
- * 0 Sticks directly map to velocity setpoints without smoothing.
- *   Also applies to vertical direction and Altitude mode.
- *   Useful for velocity control tuning.
- * 3 Sticks map to velocity but with maximum acceleration and jerk limits based on
- *   jerk optimized trajectory generator (different algorithm than 1).
- * 4 Sticks map to acceleration and there's a virtual brake drag
+ * "Direct velocity"
+ * Sticks directly map to velocity setpoints without smoothing.
+ * Also applies to vertical direction and Altitude mode.
+ * Useful for velocity control tuning.
+ * "Smoothed velocity"
+ * Sticks map to velocity but with maximum acceleration and jerk limits based on
+ * jerk optimized trajectory generator (different algorithm than 1).
+ * "Acceleration based"
+ * Sticks map to acceleration and there's a virtual brake drag
```

Comment:
The rendering of params is poor now, and this won't help. Perhaps use a bullet - it will highlight the values a bit now, and in future if we do markdown descriptions this will look better.

```suggestion
 * - "Direct velocity":
 *   Sticks directly map to velocity setpoints without smoothing.
 *   Also applies to vertical direction and Altitude mode.
 *   Useful for velocity control tuning.
 * - "Smoothed velocity":
 *   Sticks map to velocity but with maximum acceleration and jerk limits based on
 *   jerk optimized trajectory generator (different algorithm than 1).
 *  - "Acceleration based":
 *  Sticks map to acceleration and there's a virtual brake drag
```

Reply:
"plan" would be a bit strong. But I'd like to.

Reply:
But either way, worth thinking about rendering because otherwise the HTML just ignores line breaks.